src/pages/TaskList/ui/TaskList.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from 'react';
+import { Box, Container } from '@mui/material';
+import { AddTask } from 'features/Addtask/AddTask';
+import {
+    tasksActions, Status, Task, getTasks,
+} from 'entities/Task';
+import { useAppDispatch, useAppSelector } from 'shared/lib/hooks/redux';
+import { generateRandomId } from 'shared/lib/helpers';
+import { MainTask } from 'widgets/MainTask/MainTask';
+
+export const TaskList = () => {
+    const dispatch = useAppDispatch();
+    const tasks = useAppSelector(getTasks);
+
+    useEffect(() => {
+        dispatch(tasksActions.getAllTasks());
+    }, [dispatch]);
+
+    const createTask = (title: string) => {
+        const randomID = generateRandomId();
+
+        const task: Task = {
+            id: randomID,
```

Comment:
you could do this to get a bit shorter form:
```
        const id = generateRandomId();

        const task: Task = {
            id,
 ```
 however in this case it doesn't even look like you need to store id in a separate variable, so it can be just:
 ```
         const task: Task = {
            id: generateRandomId(),
 ```

Reply:
Fixed